docs/conceptual/runtime/weight.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Transaction Weight
+---
+
+When transactions are submitted to a blockchain, they are executed by the nodes
+in the network. To be economically sustainable, nodes charge a fee to execute a
+transaction that must be covered by the submitter of the transaction. The cost
+to execute transactions could vary over orders of magnitude, and thus Substrate
+provides a flexible mechanism for characterizing the total, presumably minimum,
+cost of a transaction in order to be included into a block.
+
+Furthermore, a number of resources in any chain can be limited by definition
```

Comment:
"limited by definition" seems weird to me. Where is such defining happening? I might suggest

```diff
-  a number of resources in any chain can be limited by definition
+  a number of resources in any chain can be physically scarce
```